modules/ROOT/Edits/modify-partner-settings.adoc

```diff
@@ -0,0 +1,62 @@
+= Modify a Partner's Settings
+
+You can modify a partner's settings including contact details, identifiers, digital certificates, acknowledgements, control numbers, and more.
+
+To modify a partner's settings:
+
+. In Partner Manager, select *Partners* in the navigation menu.
+. Click the partner.
+. Change any of the following settings:
```

Comment:
You typically do not want a table within a task...especially a table that contains substeps.